.vscode/launch.json

```diff
@@ -33,23 +34,23 @@
 			"preLaunchTask": "npm"
 		},
 		{
-			"name": "Launch Integration Tests",
+			"name": "Launch Extension Tests",
 			"type": "extensionHost",
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
 			// the workspace path should be GOPATH
 			"args": [
 				"--extensionDevelopmentPath=${workspaceFolder}",
-				"--extensionTestsPath=${workspaceFolder}/out/test",
+				"--extensionTestsPath=${workspaceFolder}/out/test/integration/index",
 				"--timeout",
 				"999999",
 			],
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": [
-				"${workspaceFolder}/out/**/*.js"
+				"${workspaceFolder}/out/test/**/*.js"
 			],
-			"preLaunchTask": "npm"
+			"preLaunchTask": "npm: watch"
```

Comment:
Why is it that we need a watch task to debug tests but only a compile task for debugging the extension?

Reply:
I mostly just copied code directly from the samples, so I'm actually not able to give a good answer for this. I noticed that they also had the "npm: watch" task in the "Launch Extension" in the sample, so I'll add it into my next PR.

Reply:
The pre launch task is usually used to ensure that the code has been built before debugging starts. For this purpose a simple `tsc` should be enough and we don't really need a watch.

I would suggest in removing the new task for watch and keep the old prelaunch task unless we come across the need to have a watch

Reply:
FWIW, this is where I copied this from: https://github.com/microsoft/vscode-extension-samples/blob/master/helloworld-test-sample/.vscode/launch.json. I'd be curious to know why `npm: watch` is used here rather than just `npm`. My best guess is that it would save time when debugging since your code would be getting recompiled as you edit?

Reply:
True, but then would multiple debug sessions spawn multiple watch sessions?